README.md

```diff
@@ -23,6 +26,9 @@ migration guide. All other pull requests will be closed without merging.
 [in English]: https://github.com/thoughtbot/paperclip/blob/master/MIGRATING.md
 [en español]: https://github.com/thoughtbot/paperclip/blob/master/MIGRATING-ES.md
 [a video]: https://www.youtube.com/watch?v=tZ_WNUytO9o
+[Kreeti]: https://www.kreeti.com/
```

Comment:
My browser gives me a cert error when I follow this link. Should we drop the `s`?

Reply:
Yeah I guess so. I've noted in the text that it's a cleartext link just in case someone has an ISP that screws with that, or is otherwise paranoid.